src/azul/indexer/aggregate.py

```diff
@@ -436,6 +448,9 @@ def _aggregate(self, aggregate: dict[str, Accumulator]) -> JSON:
         for k, accumulator in aggregate.items():
             if accumulator is not None:
                 result[k] = accumulator.get()
+                if accumulator.dropped > 0:
+                    log.warning('Values were dropped %d times while aggregating %r',
+                                accumulator.dropped, k)
```

Comment:
The log message should indicate the entity type. Ideally, we should add an instance attribute called `entity_type` to the `EntityAggregator` class, and supply a value for it in the constructor for that class. That's going to be fairly invasive and if it gets to complicated we can just log the class name here.